### ISSUE TITLE
Github commit not appearing on my profile

### ISSUE THREAD

**Issue by Ajeey** [enkidevs/commit#5]
Hi,
My github email is "ajey.charantimath@gmail.com" and I am logged on to enki app with the same email. At the end of each exercise, I allow the app to commit to github.
However I do not see the commits on my github profile. What am I missing?

**Comment by kbrsh** [comment]
Hey,
When you commit your progress for github, Enki doesn't create a repo on your behalf, they are all committed on **this repo**. They will show up as contributions on your profile, and here are the contributions you have made:
[https://github.com/enkidevs/commit/commits?author=Ajeey](https://github.com/enkidevs/commit/commits?author=Ajeey)

**Comment by mathieudutour** [comment]
If they do not appear, look at this: https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/

tl;dr: you might need to star the repo for the contributions to appear on your profile
